CONTRIBUTING.md

```diff
@@ -23,16 +23,16 @@ Contributing code
 
 If you'd like to contribute code to k6, this is the basic procedure. Make sure to follow the [style guide](#style-guide) described below.
 
-1. Find an issue you'd like to fix. If there is none already, or you'd like to add a feature, please open one and we can talk about how to do it.  Out of respect for your time, please start a discussion regarding any bigger contributions either in a GitHub Issue, in the community forums or in the `#contributors` channel of the k6 slack **before** you get started on the implementation.
+1. Find an issue you'd like to fix. If there is none already, or you'd like to add a feature, please open one, and we can talk about how to do it.  Out of respect for your time, please start a discussion regarding any bigger contributions either in a GitHub Issue, in the community forums or in the `#contributors` channel of the k6 slack **before** you get started on the implementation.
   
    
-   Remember, there's more to software development than code; if it's not properly planned, stuff gets messy real fast. 
+   Remember, there's more to software development than code; if it's not properly planned, stuff gets messy real fast.
 
 2. Create a fork and open a feature branch - `feature/my-cool-feature` is the classic way to name these, but it really doesn't matter.
 
 3. Create a pull request!
 
-4. Sign the [Contributor License Agreement](https://cla-assistant.io/loadimpact/k6) (the process is integrated with the pull request flow through cla-assistant.io)
+4. Sign the [Contributor License Agreement](https://cla-assistant.io/grafana/k6) (the process is integrated with the pull request flow through cla-assistant.io).
```

Comment:
I thought about changing this the other day with some of the other changes, but not sure this is what we want just yet. They do seem to cover different shared repositories.

Reply:
But check [this example](https://github.com/grafana/k6/pull/2416#issuecomment-1061077979), it seems like the new one is what the contributor should sign, no? :thinking:

Reply:
It definitely feels like that's what it should be, I guess I just had hesitation initially. 🤷